tic-tac-toe/src/app/components/textInput/index.js

```diff
@@ -1,8 +1,8 @@
 import React, { Fragment } from 'react';
 
-export const CustomInput = props => (
+export const textInput = props => (
   <Fragment>
-    <label>{props.label}</label>
+    <label htmlFor="label">{props.label}</label>
```

Comment:
The `for` attribute of a html label should have the name of the input it's labeling.